tokio-threadpool/src/builder.rs

```diff
@@ -41,7 +41,6 @@ use futures2;
 /// use std::time::Duration;
 ///
 /// # pub fn main() {
-/// // Create a thread pool with default configuration values
```

Comment:
I found this comment confusing. It seems to imply that the example shows how the builder is configured by default. However, this comment applies only to `Builder::new()`, while the chained methods do custom configuration (which is different from default).

Perhaps a different sentence would work better, but due to lack of ideas I've decided to simply remove it. :)

Reply:
👍